yml/hsc.yaml

```diff
@@ -684,6 +690,18 @@ tables:
     description: Set to 1 for any fatal failure. Measured on g-band.
     mysql:datatype: BOOLEAN
     fits:tunit:
+  - name: g_sizeExtendedness
+    "@id": "#Object.g_sizeExtendedness"
+    datatype: double
+    description: Measure of likelihood to be a galaxy. Measured on g-band.
```

Comment:
Should these all say "Moments-based estimate of likelihood to be a galaxy. Measured on g-band." etc?